components/script/script_thread.rs

```diff
@@ -1395,6 +1401,7 @@ impl ScriptThread {
             image_cache: self.image_cache.clone(),
             content_process_shutdown_chan: content_process_shutdown_chan,
             layout_threads: layout_threads,
+            paint_time_metrics: self.paint_time_metrics.clone(),
```

Comment:
It's not clear to me that we want to clone existing metrics here - this is creating a new layout thread for a subframe of this page, which will be loading an entirely separate document. I think we should be creating a new value, and if we still need it later we should store it in InProgressLoad.